modules/andes-core/broker/src/main/java/org/wso2/andes/server/cluster/coordination/rdbms/RDBMSAgent.java

```diff
@@ -163,7 +163,9 @@ public void addMessageId(String queueName, long messageId) throws AndesException
 		try {
 			andesContextStore.addMessageId(queueName, messageId);
 		} catch (AndesDataIntegrityViolationException ignore) {
-			//Primary key violation exception ignored
+			//Same message id can be added to list when slots are overlapped. In RDBMS slot store
+			//composite primary key of queue name and message id have been used to avoid duplicates.
```

Comment:
We ignore the exception because, we have forced database to not accept duplicate message ids, In hazelcast implementation, treeset has been used for this and it will ignore duplicates same manner.